app/views/vacancies/_vacancies_for_map.json.jbuilder

```diff
@@ -1,7 +1,7 @@
-json.array! vacancies.select(&:school_geolocation) do |vacancy|
+json.array! vacancies.select do |vacancy|
```

Comment:
Should this be filtering by ones that have `schoool.geolocation? == true`?

Reply:
Good catch. I dunno what made me delete that line. That said, I don't even thing we use this partial, so I've binned it

Reply:
:+1: